src/app/components/date-range-select/date-range-select.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { DateAdapter } from '@angular/material/core';
+import { Moment } from 'moment';
+import { DateSelectService } from 'src/app/services/date-select.service';
+
+@Component({
+  selector: 'app-date-range-select',
+  templateUrl: './date-range-select.component.html',
+  styleUrls: ['./date-range-select.component.scss']
+})
+export class DateRangeSelectComponent implements OnInit {
+  startDateCtrl = new FormControl(new Date());
+  endDateCtrl = new FormControl(new Date());
+  minDate = new Date(2020, 0, 1);
+  maxDate = new Date(Date.now());
+  isOpen = false;
+
+  constructor(private _adapter: DateAdapter<Moment>,private dateSelectService: DateSelectService) {
```

Comment:
Tbh I only added moment.js right at the end because the material datepicker's default format is the US version, so MM/DD/YYYY. And only introduced moment because its required to use the `MatMomentDateModule` which supplies the `DateAdapter` which has locale changes built in. The default material `MatNativeDateModule` only has the Us locale.

One solution could be to write our own dateAdapter, and use [Date.toLocaleDateString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString). Otherwise just ignore it and leave it as the default material locale.

Reply:
Actually looking at it I'm keen to just use the moment version and switch dates to moments. Can change to another implementation if we ever have issues, especially since using  the moment adapter is part of angular.